README.md

```diff
@@ -68,28 +68,53 @@ Get the repository using `go get`:
 ```bash
 go get github.com/codilime/floodgate
 ```
-In its current development state Floodgate checks if a `config.yaml` is present in the root directory.
-Navigate to the directory:
+This will download, compile and install Floodgate to `$GOPATH/bin`. If this directory is present in your `$PATH`, the `floodgate` command will be available from any directory.
+
+Create a `config.yaml` file. An example one can be found in `examples/config.yaml`.
+Floodgate checks if a config file is present in:
 ```bash
-cd $GOROOT/github.com/codilime/floodgate
+$HOME/.config/floodgate/config.yaml
 ```
-Create a `config.yaml` file. An example config can be found in `example.yaml`.
-Additionally, you can find example resources in the `resources` directory.
+You can also specify a path to the config file using the `--config` flag.
+
+Additionally, you can find example resources in the `examples/resources` directory.
 
-Execute `go build` and run the binary:
+**Note:** To allow Floodgate to control pipeline templates apply those commands to your environment:
 ```bash
-go build && ./floodgate
+hal config features edit --pipeline-templates true
+hal config features edit --managed-pipeline-templates-v2-ui true
```

Comment:
This line is not needed by Floodgate. It is required to see pipeline templates in UI only :)

Reply:
Ack

Reply:
Can you maybe add information that this part is optional and used only by UI?

Reply:
Done